account_move_name_sequence/models/account_move.py

```diff
@@ -15,6 +15,16 @@ class AccountMove(models.Model):
     sequence_prefix = fields.Char(compute=False)
     sequence_number = fields.Integer(compute=False)
 
+    _sql_constraints = [
+        (
+            "name_state_diagonal",
+            """CHECK((COALESCE(name, '') NOT IN ('/', '') AND state='posted') OR
```

Comment:
I think this condition may be simplified to use two terms instead of three:

```suggestion
            "CHECK((COALESCE(name, '') NOT IN ('/', '') OR state!='posted')"
```